features/all/phpcr_query_update.feature

```diff
@@ -153,3 +153,30 @@ Feature: Execute a a raw UPDATE query in JCR_SQL2
             | UPDATE [nt:unstructured] mixin_foo('bar') |
             | UPDATE [nt:unstructured] APPLY mixin_foo('bar') |
             | UPDATE [nt:unstructured] mixin_foo'bar') |
+
+    Scenario Outline: Execute update query with expressions
+        Given I execute the "<query>" command
+        Then the command should not fail
+        And I save the session
```

Comment:
Having ``When`` parts after the ``Then`` parts looks weird. The recommendation is to stick to this workflow (you can have multiple steps in each section if needed):

- GIVEN: initialize the state of the project (``Given a node "foobar" has been created``)
- WHEN: perform an action (``When I execute a command``)
- THEN: assert something (``Then it should fail``)

A good rule to identify where each step belongs is to read them:

- When steps should always use an action word (otherwise you cannot perform an action)
- Then steps will generally use ``should`` (but it may depend of your convention)
- Given steps will either use past sentences with action words (describing what happened previously) or descriptive verbs (``Given there is ...``) depending on your convention